cmd/limactl/start.go

```diff
@@ -529,3 +529,55 @@ func startBashComplete(cmd *cobra.Command, _ []string, _ string) ([]string, cobr
 	compTmpl, _ := bashCompleteTemplateNames(cmd)
 	return append(compInst, compTmpl...), cobra.ShellCompDirectiveDefault
 }
+
+// startWithAsking starts the instance after a confirmation prompt.
+//
+// If newInst is set to true, this function creates the instance too.
+//
+// If mandatory is set to true, and the answer to the confirmation prompt is No,
+// the function returns an error that contains an instruction to start the instance manually.
```

Comment:
The mandatory flag feels to complex. If we return a sentinel error like ErrUserCanceled the caller can print a helpful message without complicating this function.

Reply:
Tried but I felt it was even more complicated